pic32/cores/pic32/WInterrupts.c

```diff
@@ -279,11 +279,7 @@ void detachInterrupt(uint8_t interruptNum)
 #ifdef PIN_INT0
 //************************************************************************
 // INT0 ISR
-#if defined(__PIC32MZXX__)
-void __attribute__((nomips16,at_vector(_EXTERNAL_0_VECTOR),interrupt(IPL4SRS))) ExtInt0Handler(void)
```

Comment:
Yes, because those attributes are redundant. They are used to place the function at a specific location and attach it to a specific interrupt. That is not needed, since the setIntVector function does it for you. With them there you get conflicts. Without them everything works happily.

All that is needed is `nomips16` to force 32-bit code, `interrupt()` to make it use `eret` instead of `jr`, and the `section` to group the interrupts in the right region. The rest is not wanted.